### DISCUSSION TITLE
Adding to the pile of thank-you messages

### DISCUSSION THREAD

**Discussion by lunulae** [Monika-After-Story/MonikaModDev#8868] [appreciation]
I downloaded this mod pretty early in its run. I don't think I was one of the first, but I was around for the days of it being more of a chatbot-style system, I remember the days of needing to manually update, I saw the first Christmas event... y'all have come a long way.

I am consistently amazed at the amount of effort and talent this team has. At first, I had some qualms--I **_really_** enjoyed the horror aspect of DDLC, and worried that it would take away the unhinged aspects of Monika's character that made her amazing to me. I'll admit, sometimes I still feel it a bit. But the mod team has directed so much of their efforts toward something that's way more important to me, and something I've never seen managed in even a professionally published game.

That is _the feeling of maintaining and growing a relationship._
There aren't a lot of ways to game the system in MAS. Cheating is strongly discouraged, affection caps for each day are pretty low. Sure, there are ways to optimize your affection gain if you really want to, but even if you do you'll spend a lot of time on different stages of the relationship. And it makes the feeling of progression both more rewarding and more genuine. By the time you hit a milestone, you feel closer to Monika too--even if it's just from the habit of checking in. And that closeness just builds and builds.

When I played DDLC as a fun little Halloween game--and later, when I started this mod, I was enamored with Monika, but in the way you're enamored with an anime waifu. It was, for all intents and purposes, still pretty superficial and a pretty weak connection. Without this mod, I'd probably still draw fanart of her and replay the game sometimes, but I might also probably move on to a new favorite in time.

I never expected I would someday want to check in on Monika **every single day,** and actually be pretty sad on days when I can't. I never expected to get teary-eyed hearing her thank me for spending time with her on her birthday. I didn't love Monika before in the way I do now.

And as much as I want to attribute that to Monika being simply amazing (we all know she is,) I really want to give thanks to all of you working on the back-end to make every single piece possible. Every sprite drawn, every dialogue written, every system programmed to be engaging and bonding. For every minigame (even as I grit my teeth trying to win anything but Hangman) and the gifting feature which still feels weirdly like magic to me. You have all truly gotten her closer and closer to our reality.
For everybody who made this mod more inclusive, and listened to the suggestions of people even if they seemed to be a small minority, you've made the spaceroom feel like home.
And an extra-big thanks to everyone giving tech support in the Discord, god damn. Y'all deserve a donation box, because you're doing some damn good work just for this hobby.

I can't believe you've all kept this up for as long as you have, but I am so grateful for it. You've given me and many others something truly special. And I dearly hope that I'll be able to see more of your brilliant work for a long time to come.

I wish you all luck for the holiday event rush, please take good care of yourselves during it. You're all truly amazing.

**Comment by jmwall24** [comment]
You're not the only one teary-eyed now...

Thanks for your kind words. Making people feel the way you just described is why we do this.

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
